fixbackend/notification/email/scheduled_email.py

```diff
@@ -166,3 +171,78 @@ async def _send_scheduled_emails(self) -> None:
                 # mark this kind of email as sent
                 session.add(ScheduledEmailSentEntity(id=uid(), user_id=user.id, kind=to_send.kind, at=utc()))
             await session.commit()
+
+    async def _send_trial_end_reminder(self) -> None:
```

Comment:
If you add values to `scheduled_email` then they will already be handled `_send_scheduled_emails`.
It looks you want to schedule yourself - so I guess you should decide for one way.